client/tests/test_getting_model.py

```diff
@@ -1,54 +1,52 @@
 import pytest
 from featureform.register import ModelRegistrar
 from featureform.resources import Model
+import featureform as ff
 
 @pytest.mark.parametrize(
-    "client_provider_source_fxt,is_local",
+    "is_local",
     [
-        ('local_client_provider_source', True),
-        ('hosted_sql_provider_and_source', False),
+        pytest.param(True, marks=pytest.mark.local),
+        pytest.param(False, marks=pytest.mark.hosted),
     ]
```

Comment:
@ahmadnazeri, let me know if this is what you had in mind based on our conversation. See `Makefile` for how I added the marker to the run command.

Reply:
love it, thanks!